data/brands/shop/department_store.json

```
@@ -364,6 +364,34 @@
         "name": "Dunnes Stores",
         "shop": "department_store"
       }
+    },
+     {
+      "displayName": "Dufry",
+      "id": "dufry-d45fb2",
+      "locationSet": {"include": [
+        "in",
+        "mx",
+        "au",
+        "ar",
+        "pe",
+        "it",
+        "br",
+        "hk",
+        "co",
+        "ru",
+        "am",
+        "tr",
+        "ch",
+        "ma",
+        "kh",
+        "ae"
+      ]},
```

```suggestion
      "locationSet": {"include": ["001"]},
```
They have some locations in the US as well, which aren't listed on their Wikipedia article. I think it's best to include the whole world as they could have locations in other countries as well.